distro/runtime/etc/services.cfg

```diff
@@ -0,0 +1,34 @@
+# This file defines required service configurations.
+# It can be overridden by user specific configurations in conf/services folder.
+
+org.eclipse.smarthome.folder:items=items
+org.eclipse.smarthome.folder:sitemaps=sitemap
+org.eclipse.smarthome.folder:rules=rules
+org.eclipse.smarthome.folder:scripts=script
+org.eclipse.smarthome.folder:persistence=persist
+org.eclipse.smarthome.folder:things=things
+
+org.eclipse.smarthome.security:option=OFF
+
+org.eclipse.smarthome.persistence:default=rrd4j
+
+org.eclipse.smarthome.chart:provider=default
+
+org.eclipse.smarthome.classicui:defaultSitemap=_default
+
+org.eclipse.smarthome.binding.sonos:opmlPartnerID=IAeIhU42
+
+org.jupnp:multicastResponsePort=0
+org.jupnp:asyncThreadPoolSize=30
+org.jupnp:threadPoolSize=15
+
+com.eclipsesource.jaxrs.connector:root=/rest
+com.eclipsesource.jaxrs.swagger.config:swagger.basePath=/rest
+com.eclipsesource.jaxrs.swagger.config:swagger.info.title=Eclipse Smarthome REST API
+
+# Configuration of scheduled thread pool sizes
+org.eclipse.smarthome.threadpool:thingHandler=3
+org.eclipse.smarthome.threadpool:discovery=3
+
+# Non-scheduled thread pools can also provide a max size
+org.eclipse.smarthome.threadpool:safeCall=3,10
```

Comment:
In this file there are some configs of bindings which may not be installed.
I think it makes sense to have them in, even if not use. So just copy binding to addons is sufficient.
It follows same pattern as used in openHAB2 distribution.
Good idea !